### PR TITLE
PHP Version change widget

### PR DESCRIPTION
**Pull request by cr0vy**
This PR will add "PHP version change" postbox to Updates-page.

This PR does not include updating all of the Updates-page into using postboxes.

### CODE REVIEW

File: modules/updates.php [hunk 1]
```diff
@@ -39,27 +74,175 @@ public static function register_scripts( $page ) {
...
+      }
+      ?>
+      <?php
+    }
+
+    public static function change_php_version_postbox() {
+      ?>
+      <p>Latest version is recommended if all plugins and theme support it. Check <a href="tools.php?page=logs_page&logfile=wp-php-compatibility.log">compatibility scan results.</a></p>
+      
+      <div id="seravo-php-version" visibility="visible">
```
[the remark this answers is not in the document]
Comment (ypcs) [reply modules/updates.php]: (But I'm OK with current implementation too...)

File: lib/updates-ajax.php [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+/* Module which handles ajax requests for the updates page */
+
+// Deny direct access to this file
+if ( ! defined('ABSPATH') ) {
+  die('Access denied!');
+}
+
+function seravo_change_php_version() {
+  $php_version_string = ( '"set \$mode php' . $_REQUEST['version'] . ';"' );
+  exec('echo ' . $php_version_string . ' | tee /data/wordpress/nginx/php_version.conf');
```
[the remark this answers is not in the document]
Comment (ypcs) [reply lib/updates-ajax.php]: Oh, and almost forgot. Please note, that nginx reads configuration files in alphabetical order (A-Z), and uses only last config line found. So, if site has (e.g.) `set $mode php7.0;` in file `/data/wordpress/nginx/set-php-version.conf`, it doesn't matter what you configure in `/data/wordpress/nginx/php_version.conf`.

**Comment by ottok** [comment]
Can you please add a screenshot to the pull request? Thanks :smile:

**Comment by ottok** [comment]
@cr0vy Please remove the "WIP" from the title when you consider this merge ready.

**Comment by cr0vy** [comment]
@ottok Here you go! 

chage-php-version-1

chage-php-version-2

chage-php-version-3

chage-php-version-4

**Comment by ottok** [comment]
"WIP" is gone. I will merge this Wed-morning unless somebody objects.

**Pull request merged**